playground/App.vue

```diff
@@ -1,13 +1,30 @@
 <template>
     <v-app>
-        <v-container fluid fill-height>
-            <vuetify-upload
+        <v-container>
+            <VuetifyUpload
                 label="Images for the slideshow"
                 :max="4"
                 can-delete
                 accept="image/*"
                 color="accent"
-            ></vuetify-upload>
+            />
```

Comment:
Is this still supposed to be here? Because now you will have 2 exactly the same Vuetify Upload's grouped together :thinking:

Reply:
Yeah i wanted to show both that possibilities worked